test/geometry/transform/test_imgwarp.py

```diff
@@ -268,6 +278,35 @@ def test_fill_padding_channels(self, device, dtype, num_channels):
 
         assert_close(img_a[:, :, :1, :1].squeeze(), fill_value.squeeze())
 
+    @pytest.mark.parametrize("align_corners", (True, False))
+    @pytest.mark.parametrize("padding_mode", ("zeros", "fill"))
+    def test_jit_script(self, device, dtype, align_corners, padding_mode):
```

Comment:
is it relevant to keep torchscript test is we go for onnx ? underhood it traces in any case

Reply:
I consider onnx to be one of the end states of scripting pipeline of torch (tensorrt is another widely used format that comes to mind), so I think it's worth keeping the test for an intermediate torch.jit.script product.

Again, that's coming from me as a user of kornia rather than a person who has insight into the project.

Reply:
sure, it's just that we have had such a base experience with torchscript along the years and prioritised to onnx recently  as we seen the most claimed format but always open to community needs.